src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -537,15 +538,19 @@ g.test('resolveTarget,usage')
     `
   Test that using a resolve target whose usage is not RENDER_ATTACHMENT is invalid for color
   attachments.
-
-  TODO: Add a control case (include vs exclude RENDER_ATTACHMENT usage)
   `
   )
+  .paramsSimple([
+    { usage: GPUConst.TextureUsage.COPY_SRC | GPUConst.TextureUsage.COPY_DST },
+    { usage: GPUConst.TextureUsage.STORAGE_BINDING | GPUConst.TextureUsage.TEXTURE_BINDING },
+    { usage: GPUConst.TextureUsage.STORAGE_BINDING | GPUConst.TextureUsage.STORAGE },
+    { usage: GPUConst.TextureUsage.RENDER_ATTACHMENT | GPUConst.TextureUsage.TEXTURE_BINDING },
+  ])
```

Comment:
note: I think just 2 cases would have been sufficient (e.g. `COPY_SRC | COPY_DST` and `COPY_SRC | COPY_DST | RENDER_ATTACHMENT`), but this is also fine

Reply:
It's likely I added unnecessary cases. Thanks!